bin/juttle-engine-client

```diff
@@ -28,7 +28,21 @@ function push_file(filename, topic) {
             bundler.bundle(filename)
                 .then(function(bundle) {
                     return new Promise(function (resolve, reject) {
-                        ws.send(JSON.stringify(bundle), resolve);
+                        ws.on('message', (data) => {
+                            data = JSON.parse(data);
+
+                            if (data.type === 'error') {
+                                reject(data.error);
+                            } else {
+                                resolve();
+                            }
+                        })
+
+                        ws.send(JSON.stringify({
+                            type: 'bundle',
+                            bundle_id: filename,
+                            bundle
```

Comment:
you're talking about the non, error response right? I'm using fancy es6 object syntax which converts `{ bundle }` to `{ bundle: bundle }`